pkg/cloudprovider/providers/gce/gce_instances.go

```diff
@@ -317,6 +327,17 @@ func (gce *GCECloud) GetAllZones() (sets.String, error) {
 	return zones, nil
 }
 
+// SetClientSet sets the ClientSet on the GCECloud object for e2e tests
+func (gce *GCECloud) SetClientSet(c clientset.Interface) {
```

Comment:
Do we still need this, if the e2e binary is still getting all the zones from the cloud provider?

Reply:
Technically not, but it might still be useful for the future?

Reply:
I would leave it out until we need it